### PR TITLE
Update Lisk SDK to v6.0.0-beta.6

### LINKED ISSUE #818: Update Lisk SDK to v6.0.0-beta.6
### Description

Update Lisk SDK to v6.0.0-beta.6.

### Acceptance Criteria

- Update SDK to v6.0.0-beta.6
- Update the genesis blocks
  - Lisk DB [bug fixes](https://github.com/LiskHQ/lisk-sdk/pull/8821/files#diff-6ed0c602abeaef5b683446326e877918e610000b299acd9484173167ceaba615)
- Update p2p ports in mainnet/testnet configs to `7667`
- Load the plugins in-memory if https://github.com/LiskHQ/lisk-sdk/issues/8786 is not fixed

### PR DESCRIPTION
**Pull request by sameersubudhi**
### What was the problem?

This PR resolves #818 #817 

### How was it solved?

- [x] Upgrade Lisk SDK to v6.0.0-beta.6
- [x] Updated genesis blocks for devnet, alphanet and betanet
- [x] Updated default P2P ports for mainnet and testnet
- [x] Configure chain-connector-plugin to load in-memory
- [x] Update Makefile to use Docker BuildKit to build docker images

### How was it tested?

Local with both devnet/betanet configs

### CODE REVIEW

File: src/commands/start.ts [hunk 1]
```diff
@@ -132,7 +132,7 @@ export class StartCommand extends BaseStartCommand {
 			app.registerPlugin(new FaucetPlugin(), { loadAsChildProcess: true });
 		}
 		if (flags['enable-chain-connector-plugin']) {
-			app.registerPlugin(new ChainConnectorPlugin(), { loadAsChildProcess: true });
+			app.registerPlugin(new ChainConnectorPlugin(), { loadAsChildProcess: false });
```
Comment (nagdahimanshu) [inline src/commands/start.ts:135]: Why this change is only for `chainConnectorPlugin`?

Comment (sameersubudhi) [reply src/commands/start.ts:135]: The child process creation times out only fir the chain connector plugin. You can find the cause [here](https://github.com/LiskHQ/lisk-sdk/issues/8786#issuecomment-1665199369).

**Review by nagdahimanshu — APPROVED** [review]
LGTM 👍

**Review by vardan10 — APPROVED** [review]
